Android/app/src/main/res/layout/activity_home.xml

```diff
@@ -42,73 +53,49 @@
             android:layout_below="@id/toolbar"
             app:layout_constraintBottom_toBottomOf="parent"
             android:orientation="vertical">
-        <LinearLayout
-                android:id="@+id/LinearLayoutWithText"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                app:layout_constraintTop_toBottomOf="@id/LinearLayout"
-                android:layout_marginTop="10dp"
-                android:layout_below="@id/LinearLayout"
-                app:layout_constraintBottom_toBottomOf="parent"
-                android:orientation="vertical">
             <LinearLayout
+                android:id="@+id/LinearLayoutWithText"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="horizontal"
-                android:layout_marginBottom="5dp"
-                android:layout_marginTop="5dp">
+                app:layout_constraintTop_toTopOf="@id/LinearLayout"
+                android:orientation="horizontal">
+                <com.google.android.material.textfield.TextInputLayout
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="5dp"
+                    android:layout_gravity="center_vertical|start"
+                    android:layout_weight="1"
+                    style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox">
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:inputType="textAutoComplete"
+                        android:textColor="@color/cardText"
+                        android:hint="Search"
+                        android:id="@+id/brewerySearchBar" />
+                </com.google.android.material.textfield.TextInputLayout>
                 <com.google.android.material.button.MaterialButton
                     android:id="@+id/filterButton"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_gravity="center_horizontal"
-                        android:gravity="right"
-                        android:layout_marginEnd="40dp"
-                    android:padding="12dp"
+                    android:layout_gravity="center_vertical|end"
+                    android:layout_marginStart="10dp"
+                    android:padding="8dp"
                     android:text="Filters"
-                    android:textColor="@color/cardview_light_background"
+                    android:textColor="@color/cardBackgroun"
```

Comment:
cardBackground?

Reply:
Yeah I just made new color themes for the app.

Reply:
you spelt it wrong though.